src/xrGame/player_hud_tune.cpp

```diff
@@ -349,32 +349,32 @@ void hud_adjust_mode_keyb(int dik)
 {
     if (pInput->iGetAsyncKeyState(SDL_SCANCODE_LSHIFT))
     {
-        if (dik == SDL_SCANCODE_KP_0)
+        if (dik == SDL_SCANCODE_KP_0 || dik == SDL_SCANCODE_0)
```

Comment:
@Xottab-DUTY is it okay to keep this change in? If you use keyboard without numpad (for instance, most 66% layout keyboards), player_hud_tune is completely unusable unless your specific keyboard supports faking numpad inputs.